### PR TITLE
Working from notebooks

### PR DESCRIPTION
**Pull request by emilyselwood**
This set of code changes allows the cube query system to pull jupyter notebooks from a git repo and use those as processes.

This replaces the old way of doing this with the cubequery-depolyment repo.

most of the code is in the git_packages.py file. There are a few notebooks added to the repo so that we have some test cases.

### CODE REVIEW

File: cubequery/api_server.py [hunk 1]
```diff
@@ -47,7 +50,8 @@ def _to_bool(input):
 )
 
 packages = [m['name'].replace("/", ".") for m in list_processes()]
-celery_app.autodiscover_tasks(packages=packages, related_name="", force=True)
+if len(packages) > 0:
```
Comment (Juanezm) [inline cubequery/api_server.py:53]: the > 0 part is redundant in this case

File: cubequery/tasks/__init__.py [hunk 2]
```diff
@@ -352,7 +369,22 @@ def check_float(s):
...
+
+    if not isinstance(param.valid, list):
+        return True
+
+    if len(param.valid) == 2:
+        pass
+    else:
+        pass
+
+    v = float(s)
```
Comment (Juanezm) [inline cubequery/tasks/__init__.py:386]: I don't understand the purpose of this function, specially the last part where whatever the len is the process 'pass' to assign the float to v variable

File: docker-compose.yml [hunk 3]
```diff
@@ -21,7 +20,17 @@ services:
       - APP_RESULT_DIR=/data/
     restart: always
     volumes:
-      - ./outputdata:/data
+      - ./redisdata:/data
+
+#  cubequery-worker:
```
Comment (Juanezm) [inline docker-compose.yml:25]: wouldn't be better to remove these lines rather than comment them out?

File: cubequery/tasks/__init__.py (hunk 2 above)
Comment (emilyselwood) [reply cubequery/tasks/__init__.py:386]: Good spot, clearly some incomplete code.

It should be checking min max if there are two entries, and its one of the values if there are more than two

File: docker-compose.yml (hunk 3 above)
Comment (emilyselwood) [reply docker-compose.yml:25]: Probably shouldn't be commented out, I disabled the worker as it was faster to test...

**Comment by james-hinton** [comment]
I tested the changes and runs fine for me

**Pull request merged**